docs/installer/lokoctl.md

```diff
@@ -20,26 +20,26 @@ These binaries can be manually downloaded and installed.
    keys](https://github.com/kinvolk/lokomotive/blob/master/docs/KEYS.md).
 
 ```console
-gpg --verify lokoctl_v0.8.0_linux_amd64.tar.gz.sig
+gpg --verify lokoctl_v0.9.0_linux_amd64.tar.gz.sig
 ```
 
 3. Unpack it
 
 ```console
-tar xvf lokoctl_v0.8.0_linux_amd64.tar.gz
+tar xvf lokoctl_v0.9.0_linux_amd64.tar.gz
 ```
 
 4. Find the lokoctl binary in the unpacked directory and move it to its desired location
 
 ```console
-mv lokoctl_v0.8.0_linux_amd64/lokoctl ~/.local/bin/lokoctl
+mv lokoctl_v0.9.0_linux_amd64/lokoctl ~/.local/bin/lokoctl
 ```
 
 5. Verify the version of `lokoctl`
 
 ```console
 lokoctl version
-v0.8.0
+v0.9.0
```

Comment:
This PR is missing the follow up commits for the new release 0.9.0 and 0.9.0-git bump.

Reply:
Yes, I just wanted to get the ball rolling with the CHANGELOG review